### ISSUE TITLE
Problem with some stuff

### ISSUE THREAD

**Issue by xUrusReal** [Cvolton/GMDprivateServer#766]
I can create levels and upload them, but I can't play them
I can't post comments, posts or anything
I am using freewebhostingarea (they didnt terminate me)

I did do connection.php
I set the data folder to chmod 777
and the link is the same amount of characters as www.boomlings.com

What do I do?

**Comment by Catover203** [comment]
Did you upload database.sql to yor sql database?
Did you edit connection.php correct with your database server?

**Comment by xUrusReal** [comment]
Yes i did everything. Playing levels magically fixed itself, but I still can't post coments or posts.
